pkg/cable/libreswan/libreswan.go

```diff
@@ -246,11 +246,10 @@ func (i *libreswan) GetConnections() ([]subv1.Connection, error) {
 }
 
 func extractSubnets(endpoint subv1.EndpointSpec) []string {
-	// Subnets
-	subnets := []string{endpoint.PrivateIP + "/32"}
+	subnets := []string{}
```

Comment:
```suggestion
	subnets := make([]string, 0, len(endpoint.Subnets))
```

Reply:
what's the benefit of that, not having to grow the slice later I guess?

Reply:
https://medium.com/vendasta/golang-the-time-complexity-of-append-2177dcfb6bad#:~:text=In%20Go%2C%20append(slice%2C,cost%20is%20O(N).

Reply:
nice. I'll let E2E run, and I'll update the patch.